docs/deployment/filesystems/plugins.md

```diff
@@ -70,7 +70,7 @@ possible across Flink core, plugins, and user code.
 
 ## File Systems
 
-All [file systems](filesystems) **except MapR** are pluggable. That means they can and should
+All [file systems]() **except MapR** are pluggable. That means they can and should
```

Comment:
I guess it makes sense to put a proper link here?

Reply:
Yes this makes sense.